apps/condo/domains/ticket/hooks/useTicketExportTask.tsx

```diff
@@ -16,6 +16,17 @@ export const useTicketExportTaskUIInterface = () => {
     const TicketExportTaskProgressDescriptionCompleted = intl.formatMessage({ id: 'tasks.TicketExportTask.progress.description.completed' })
     const router = useRouter()
 
+    const tryToDownloadFile = (taskRecord: TicketExportTask) => {
+        if (window) {
```

Comment:
If this is check for SSR please use type safe pattern `typeof window !== 'undefined'`

Reply:
I think, this check is redundant because on server side this never will be executed